Release documentation improvements (#5103)
elastic/cloud-on-k8s: docs/operating-eck/operator-config.asciidoc

|metrics-port |0 |Prometheus metrics port. Set to 0 to disable the metrics endpoint.
|namespaces |"" |Namespaces in which this operator should manage resources. Accepts multiple comma-separated values. Defaults to all namespaces if empty or unspecified.
|operator-namespace |"" |Namespace the operator runs in. Required.

> |set-default-security-context |true | Enables adding a default Pod Security Context to Elasticsearch Pods in Elasticsearch `8.0.0` and above. The default added sets `fsGroup` field to `1000` to match Elasticsearch container default UID. This behavior might not be appropriate for OCP and PSP-secured Kubernetes clusters, so it can be disabled.

Reviewer: ```suggestion
|set-default-security-context |true | Enables adding a default Pod Security Context to Elasticsearch Pods in Elasticsearch `8.0.0` and above. `fsGroup` is set to `1000` by default to match Elasticsearch container default UID. This behavior might not be appropriate for OpenShift and PSP-secured Kubernetes clusters, so it can be disabled.
```
